client/landing/gutenboarding/hooks/use-on-site-creation.ts

```diff
@@ -107,10 +108,9 @@ export default function useOnSiteCreation() {
 					resetOnboardStore();
 					setSelectedSite( newSite.blogid );
 
-					const redirectionUrl =
-						selectedPlan.getStoreSlug() === PLAN_ECOMMERCE
-							? `/checkout/${ newSite.site_slug }?preLaunch=1&isGutenboardingCreate=1`
-							: `/checkout/${ newSite.site_slug }?preLaunch=1&isGutenboardingCreate=1&redirect_to=%2Fblock-editor%2Fpage%2F${ newSite.site_slug }%2Fhome`;
+					const redirectionUrl = selectedPlan.isPublicSite
```

Comment:
Forcing a redirect from checkout and the site being created as public or not are separate things. At some point we may want also the BLOGGER plan to follow the default redirect rules set in `/checkout` but it won't be created as public.